### DISCUSSION TITLE
Looking for Maintainers and Leaders

### DISCUSSION THREAD

**Discussion by Philip-Clark** [club-40/odin-pantry#7] [General]
We all agree there needs to be some experienced people at the front of this project. So even if you don't have the time/desire to participate in the actual creation of this project, if you have the skills and desire to be a maintainer or lead a part of the project, please comment below. Also, if you have seen someone say they are interested in this, please comment below and/or make them aware of this discussion.

**Comment by Philip-Clark** [comment]
I'm going to offer myself as a wild card.
I am willing to pick up any slack there is in this department, but I am also willing to give up these responsibilities if someone else is truly excited and/or experienced in these areas.

**Comment by Ananth-Adhikarla** [comment]
i can volunteer but no experience in maintaining that, I can be the last pick i dont mind

**Comment by mmxz-mmxz** [comment]
Volunteering as a maintainer - will most likely not be contributing to code, but am available for mentorship, project planning, task creation/delegation, and review. I may have less time to dedicate to this project than others though, so if you are looking for someone more involved let me know!

**Comment by JoshDevHub** [comment]
Hi,

I can volunteer to help maintain the backend stuff. A bit unsure how much code I'll be able to write, but I think I'm the most experienced Ruby/Rails person willing to volunteer for this role (feel free to correct me if I'm wrong @ any other rubyist on this project). Glad to help with review and organization stuff.

**Comment by crespire** [comment]
Happy to help with maintenance tasks like labelling things, etc! Would definitely be in a supporting role, rather than leading.

**Comment by Lofty-Brambles** [comment]
I do not mind helping out with the project in reviewing and merging PR's as necessary.
I can help out with ts, and the general javascript stuff as and when necessary, and I'll willing to put in some work with the code.

**Comment by 01zulfi** [comment]
Heyy, I'm willing to help out in GitHub organisation stuff as well as reviewing/merging.

**Comment by Aureatus** [comment]
I also don't mind reviewing and merging PR's as necessary like lofty, I can also help out with ts, react and react native if that ends up being an avenue we pursue. Been using github projects with issues to work on my current app for a while now, but not really done anything special with it so I don't think I have any unique perspective/help to give there.

**Comment by Philip-Clark** [comment]
We now have around 7 Maintainers.
I think we have a good balance of experienced maintainers and less experienced, but eager maintainers.
This is great news, thanks for volunteering everyone!

Note: There is the option of creating 2 teams within the Maintainer team. Maybe one for experienced and one for learning. I don't find this necessary at the moment though.
